### PR TITLE
Added command to create block container

### PR DESCRIPTION
**Pull request by core23**
I am targetting this branch, because it's a new feature.

Rebase of #629

## Changelog

```markdown
### Added
- Added new command to create block container for all pages
```

## Subject

This command allows you to create new blocks for all pages.

### CODE REVIEW

**Comment by core23** [comment]
Ping @greg0ire @OskarStark

File: Command/CreateBlockContainerCommand.php [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+        $this->setDescription('Creates a block container in all pages for specified template code');
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $templateCode = $input->getArgument('templateCode');
```
Comment (greg0ire) [inline Command/CreateBlockContainerCommand.php]: One time variable

File: Command/CreateBlockContainerCommand.php [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        $blockCode = $input->getArgument('blockCode');
+        $blockName = $input->getArgument('blockName');
```
Comment (greg0ire) [inline Command/CreateBlockContainerCommand.php]: One time variable

File: Tests/Command/CreateBlockContainerCommandTest.php [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    protected $blockManager;
+
+    protected $container;
+
+    public function setUp()
+    {
+        $this->blockInteractor = $blockInteractor = $this->getMockBuilder('Sonata\PageBundle\Entity\BlockInteractor')
+            ->disableOriginalConstructor()
+            ->setMethods(array('createNewContainer'))
+            ->getMock();
```
Comment (greg0ire) [inline Tests/Command/CreateBlockContainerCommandTest.php]: Please use Prophecy

**Comment by core23** [comment]
Switched to prophecy

File: Command/CreateBlockContainerCommand.php [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+namespace Sonata\PageBundle\Command;
+
+use Sonata\PageBundle\Entity\BlockInteractor;
+use Sonata\PageBundle\Model\PageInterface;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Creates block containers for all pages.
```
Comment (OskarStark) [inline Command/CreateBlockContainerCommand.php]: @author missing

File: Command/CreateBlockContainerCommand.php [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+                'page' => $page,
+                'code' => $blockCode,
+            ));
+
+            $page->addBlocks($block);
+
+            $pageManager->save($page);
+        }
+
+        $output->writeln(sprintf('Don\'t forget to add block <comment>%s</comment> into your <comment>sonata_page.yml</comment>', $blockCode));
```
Comment (OskarStark) [inline Command/CreateBlockContainerCommand.php]: please linebreak this

File: Command/CreateBlockContainerCommand.php (hunk 4 above)
Comment (core23) [inline Command/CreateBlockContainerCommand.php]: Fixed

**Comment by core23** [comment]
Added a short documentation

**Comment by greg0ire** [comment]
LGTM

**Comment by OskarStark** [comment]
Thank you @core23 👍

**Pull request merged**